munimap/static/js/draw-layer-config.js

```diff
@@ -55,7 +55,7 @@ angular.module('munimapDraw', [
         function(LayersService, DrawService, DefaultStyle, ColorpickerPalette, munimapConfig,
             StyleSchema, StyleSchemaFormOptions, LabelSchema, LabelSchemaFormOptions, DrawIconsPrefix) {
 
-            if(munimapConfig.components.draw !== undefined || munimapConfig.components.geoeditor !== undefined) {
+            if(munimapConfig.components.draw !== undefined || munimapConfig.components.geoeditor === true) {
```

Comment:
why not simply `!!munimapConfig.components.draw || !!munimapConfig.components.geoeditor`

Reply:
don't know. the other version just a little more verbose.

Reply:
```suggestion
            if(!!munimapConfig.components.draw || !!munimapConfig.components.geoeditor) {
```